src/components/Thanks/Badges/FirstScreen.vue

```diff
@@ -2,9 +2,36 @@
 	<div
 		class="tw-relative tw-text-center md:tw-rounded-t hide-for-print"
 	>
+		<!-- Opt In module -->
+		<opt-in-module
+			v-if="!isOptedIn"
+			:loans="loans"
+			:is-guest="isGuest"
+			:number-of-badges="numberOfBadges"
+			class="print:tw-hidden opt-in-module"
+		/>
 		<div
 			class="tw-pt-4 tw-pb-5 bg-gradient"
 		>
+			<div
```

Comment:
So I do think we can remove this avatar, keeping the changes to this file minimal. The general hope was just to allow a way for users to opt-in from the old badge TY page, so it doesn't skew results with the new TY page that already has the opt-in module.